modular_stonehedge/licensed-death-rattler/Death-Rattler/code/Death-Rattler.dm

```diff
@@ -0,0 +1,158 @@
+	desc = "Dr. V's nutritional suppliment; normal people eat and drink, but true champions NOURISH."
+	volume = 135
+	amount_per_transfer_from_this = 135
+	list_reagents = list(/datum/reagent/consumable/honey = 30, /datum/reagent/consumable/nutriment/vitamin = 5, /datum/reagent/water = 100) // as much water as a bucket so full hydrate. enough honey to IF good rng rolls get 30hp. 33% chance to decay into sugar, sugar has a 33% chance to decay into nutrients. so saturation varies.
+
+/datum/reagent/drug/snekbt
+	name = "Snake Bite"
+	description = "I'm not the one who's so far away..."
+	color = "#00FF00"
+	metabolization_rate = 0.1
+	overdose_threshold = 6
+
+/datum/reagent/drug/snekbt/on_mob_life(mob/living/carbon/M)
+	M.apply_status_effect(/datum/status_effect/buff/snekbt)
+	if(M.has_status_effect(/datum/status_effect/debuff/sleepytime))
+		M.remove_status_effect(/datum/status_effect/debuff/sleepytime)
+	if(M.has_flaw(/datum/charflaw/addiction/junkie))
+		M.sate_addiction()
+	..()
+
+
+/datum/reagent/drug/snekbt/on_mob_metabolize(mob/living/M)
+	..()
+	M.update_body_parts_head_only()
+	M.overlay_fullscreen("snekbt", /atom/movable/screen/fullscreen/snekbt)
+
+/atom/movable/screen/fullscreen/snekbt
+	icon_state = "curse1"
+	plane = BLACKNESS_PLANE
+	layer = AREA_LAYER
+	blend_mode = 0
+	alpha = 255
+	show_when_dead = FALSE
+
+/datum/reagent/drug/snekbt/on_mob_end_metabolize(mob/living/M)
+	M.rogstam_add(-2500)
+	M.rogfat_add(2500) // crash you fucking junkie.
+	..()
+	. = 1
+
+/atom/movable/screen/fullscreen/bittensm/Initialize()
+	..()
+	filters += filter(type="angular_blur",x=5,y=5,size=1)
+
+/datum/reagent/drug/snekbt/overdose_start(mob/living/M, can_overdose = TRUE)
+	if(HAS_TRAIT(M, TRAIT_CRACKHEAD))// boathian bullshit isnt going to help you here. its a toxin not a drug
+		can_overdose = TRUE
+	to_chat(M, span_danger("you really..... really shouldnt of done that..."))
+	M.ForceContractDisease(new /datum/disease/heart_failure(), FALSE, TRUE)
+	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "[type]_overdose", /datum/mood_event/overdose, name)
+
+/datum/reagent/drug/snekbt/overdose_process(mob/living/M)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5)
+	ADD_TRAIT(M, TRAIT_PARALYSIS_L_ARM, TRAIT_GENERIC)
+	ADD_TRAIT(M, TRAIT_PARALYSIS_L_LEG, TRAIT_GENERIC)
+	M.adjustOrganLoss(ORGAN_SLOT_HEART, 5)
+	M.adjustOxyLoss(5*REM, 0)
+	..()
+
+/datum/status_effect/buff/snekbt
+	id = "Snake Bite"
+	alert_type = /atom/movable/screen/alert/status_effect/buff/druqks
+	effectedstats = list("strength" = 2, "constitution" = 2, "speed" = -3)
+	duration = 31+13+22.6 SECONDS
+
+/datum/stressevent/snekbt
+	stressadd = -15
+	desc = span_blue("How many will you take with you?...")
+	timer = 31+13+22.6 SECONDS
+
+
+/datum/status_effect/buff/snekbt/on_apply()
+	. = ..()
+	if(owner?.client)
+		if(owner.client.screen && owner.client.screen.len)
+			var/atom/movable/screen/plane_master/game_world/PM = locate(/atom/movable/screen/plane_master/game_world) in owner.client.screen
```

Comment:
```suggestion
			var/atom/movable/screen/plane_master/PM = locate(/atom/movable/screen/plane_master/game_world) in owner.client.screen
```
`/atom/movable/screen/plane_master/game_world_fov_hidden` and `/atom/movable/screen/plane_master/game_world_above` are not subtypes of `/atom/movable/screen/plane_master/game_world`